api/swagger.yaml

```diff
@@ -243,6 +243,9 @@ definitions:
       ReadOnly:
         description: "Whether the mount should be read-only."
         type: "boolean"
+      Consistency:
+        description: "The consistency requirement for the mount: `default`, `consistent`, `cached`, or `delegated`."
```

Comment:
It probably doesn't need to be documented other than what the default actually is vs a value of "default".